doc/tutorials/platformer/app/lib/overlays/heart.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:ember_quest/ember_quest.dart';
+import 'package:flame/components.dart';
+
+enum HeartState {
+  available,
+  unavailable,
+}
+
+class HeartHealthComponent extends SpriteGroupComponent<HeartState>
+    with HasGameRef<EmberQuestGame> {
+  late int _heartNumber;
+
+  HeartHealthComponent({
+    required int heartNumber,
+    required Vector2? position,
+    required Vector2? size,
+    Vector2? scale,
+    double? angle,
+    Anchor? anchor,
+    int? priority,
+  }) : super(
+          position: position,
+          size: size,
+          scale: scale,
+          angle: angle,
+          anchor: anchor,
+          priority: priority,
+        ) {
+    _heartNumber = heartNumber;
+  }
+
+  @override
+  Future<void> onLoad() async {
+    await super.onLoad();
+    final availableSprite = await gameRef.loadSprite(
+      'heart.png',
+      srcSize: Vector2.all(32),
+    );
+
+    final unavailableSprite = await gameRef.loadSprite(
+      'heart_half.png',
+      srcSize: Vector2.all(32),
+    );
+
+    sprites = {
+      HeartState.available: availableSprite,
+      HeartState.unavailable: unavailableSprite,
+    };
+
+    current = HeartState.available;
+  }
+
+  @override
+  Future<void> update(double dt) async {
```

Comment:
This is not the correct override of `update` (it's not an async method)

Reply:
Fixed.  Not certain how that got there, must have used the wrong auto complete.